app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -19,6 +19,7 @@ import org.mozilla.focus.utils.UrlUtils
 import java.util.UUID
 
 private const val PREF_HOME_TILES = "homeTiles"
+private const val BUNDLED_SITES_BLACKLIST = "blacklist"
```

Comment:
nit, be explicit that this is of ids.

BUNDLED_SITES_IDS_BLACKLIST
and
blacklist_ids

That way, anyone who is looking at the blacklist variable names knows what the list is composed of (ids), rather than having to guess, or find other code that actually adds items to the blacklist to figure that out.